app/__init__.py

```diff
@@ -1,7 +1,8 @@
 # senPyTweet web-layer API
 
 from flask import Flask, request, jsonify
-from app.sen_py import SenPy
+from app.sen_py import SenPy, clean_tweet
```

Comment:
Do you need to import `clean_tweet` here?

Reply:
No I don't,  thanks for catching. I abandoned `clean_tweet` because it was hurting model performance. Still not sure why that is, but I think it's due to spaCy's vectorization/embedding. Will investigate further.